src/xdist/newhooks.py

```diff
@@ -55,3 +55,55 @@ def pytest_xdist_node_collection_finished(node, ids):
 @pytest.mark.firstresult
 def pytest_xdist_make_scheduler(config, log):
     """ return a node scheduler implementation """
+
+
+@pytest.mark.trylast
```

Comment:
```suggestion
@pytest.hookspec(trylast=True, firstresult=True)
```

Besides using the new syntax, I think we should add `firstresult` here to make it clear in the API that we will only use the first result from the hook.